examples/cleanrl_examples/ppo_atari_envpool.py

```diff
@@ -40,171 +41,171 @@ def parse_args():
   # fmt: off
   parser = argparse.ArgumentParser()
   parser.add_argument(
-    '--exp-name',
+    "--exp-name",
     type=str,
     default=os.path.basename(__file__).rstrip(".py"),
-    help='the name of this experiment'
+    help="the name of this experiment"
   )
   parser.add_argument(
-    '--gym-id',
+    "--gym-id",
     type=str,
     default="Pong-v5",
-    help='the id of the gym environment'
+    help="the id of the gym environment"
   )
   parser.add_argument(
-    '--learning-rate',
+    "--learning-rate",
     type=float,
     default=2.5e-4,
-    help='the learning rate of the optimizer'
+    help="the learning rate of the optimizer"
   )
   parser.add_argument(
-    '--seed', type=int, default=1, help='seed of the experiment'
+    "--seed", type=int, default=1, help="seed of the experiment"
   )
   parser.add_argument(
-    '--total-timesteps',
+    "--total-timesteps",
     type=int,
     default=10000000,
-    help='total timesteps of the experiments'
+    help="total timesteps of the experiments"
   )
   parser.add_argument(
-    '--torch-deterministic',
+    "--torch-deterministic",
     type=lambda x: bool(strtobool(x)),
     default=True,
-    nargs='?',
+    nargs="?",
     const=True,
-    help='if toggled, `torch.backends.cudnn.deterministic=False`'
+    help="if toggled, `torch.backends.cudnn.deterministic=False`"
   )
   parser.add_argument(
-    '--cuda',
+    "--cuda",
     type=lambda x: bool(strtobool(x)),
     default=True,
-    nargs='?',
+    nargs="?",
     const=True,
-    help='if toggled, cuda will be enabled by default'
+    help="if toggled, cuda will be enabled by default"
   )
   parser.add_argument(
-    '--track',
+    "--track",
     type=lambda x: bool(strtobool(x)),
     default=False,
-    nargs='?',
+    nargs="?",
     const=True,
-    help='if toggled, this experiment will be tracked with Weights and Biases'
+    help="if toggled, this experiment will be tracked with Weights and Biases"
   )
   parser.add_argument(
-    '--wandb-project-name',
+    "--wandb-project-name",
     type=str,
-    default="cleanRL",
+    default="ppo-implementation-details",
     help="the wandb's project name"
   )
   parser.add_argument(
-    '--wandb-entity',
+    "--wandb-entity",
     type=str,
     default=None,
     help="the entity (team) of wandb's project"
   )
   parser.add_argument(
-    '--capture-video',
+    "--capture-video",
     type=lambda x: bool(strtobool(x)),
     default=False,
-    nargs='?',
+    nargs="?",
     const=True,
-    help='whether to capture videos of the agent performances '
-    '(check out `videos` folder)'
+    help="weather to capture videos of the agent performances "
+    "(check out `videos` folder)"
   )
```

Comment:
unused args?

Reply:
Yeah, let's remove it. Recording the video will slow down the throughput considerably I think.